app/src/main/java/eu/jonahbauer/qed/model/parser/EventParser.java

```diff
@@ -174,6 +174,7 @@ static Double parseCost(@NonNull Element element) {
             String match = matcher.group(1);
             assert match != null;
 
+            match = match.replaceAll("\\.","");
```

Comment:
Using `String#replace(CharSequence, CharSequence)` would avoid the additional overhead of RegExp parsing.